index.html

```diff
@@ -1,6 +1,7 @@
 ---
 layout: default
 title: minuscode
+description: It’s less about code and more about quality, a product, an idea, a vision. We can help you build it.
```

Comment:
unsure about the copy on this.

i would drop "We can help you build it." so as not to come on too strong but that's a non binding opinion